styles/style.css

```
@@ -0,0 +1,242 @@
+@font-face {
+  font-family: "PT sans";
+  font-style: normal;
+  font-weight: 400;
+  src: url("../fonts/ptsans-400.woff2") format("woff2"),
+    url("../fonts/ptsans-400.woff") format("woff");
+  font-display: swap;
+}
+@font-face {
+  font-family: "PT sans";
+  font-style: normal;
+  font-weight: 400;
+  src: url("../fonts/ptsans-400.woff2") format("woff2"),
+    url("../fonts/ptsans-400.woff") format("woff");
+  font-display: swap;
+}
+@font-face {
+  font-family: "PT sans";
+  font-style: normal;
+  font-weight: 700;
+  src: url("../fonts/ptsans-700.woff2") format("woff2"),
+    url("../fonts/ptsans-700.woff") format("woff");
+  font-display: swap;
+}
+@font-face {
+  font-family: "PT sans";
+  font-style: normal;
+  font-weight: 700;
+  src: url("../fonts/ptsans-700.woff2") format("woff2"),
+    url("../fonts/ptsans-400.woff") format("woff");
+  font-display: swap;
+}
+
+body {
+  font-family: "PT sans";
+  font-size: 18px;
+  font-weight: normal;
+  font-style: normal;
+  line-height: 23px;
+  background-color: #ffffff;
+  color: #000000;
+}
+
+/*Pages*/
+
+.navigation-link {
+  font-family: PT Sans;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 20px;
+  line-height: 26px;
+  color: #000000;
+  text-decoration: none;
+  text-transform: uppercase;
+}
+.main {
+  background-color: #e5e5e5;
+  color: #000000;
+  background-image: url("../img/index-background.jpg");
+  background-size: 100% auto;
+  background-repeat: no-repeat;
+}
+.form {
+  background-color: #e5e5e5;
+  color: #ffffff;
+  background-image: url("../img/backgroundform.jpg");
+  background-size: 100% auto;
+  background-repeat: no-repeat;
+}
+.catalog-products {
+  background-color: #e5e5e5;
+  color: #ffffff;
+  background-image: url("../img/catalog-background.jpg");
+  background-size: 100% auto;
+  background-repeat: no-repeat;
+}
+.advantages-item-white {
+  background-color: #83b3d3;
+  color: #ffffff;
+}
+.advantages-describe-list {
+  background-color: #ffffff;
```

```suggestion
  background-color: #ffffff;
  color: #333333;
```

И отдельно заголовкам поставить черный цвет. Или по другому классы организовать